mi/match.c

```diff
@@ -749,8 +821,49 @@ dtheight(Dtree *dt)
 	return m+1;
 }
 
+static size_t
+refcntsum(Dtree *dt)
```

Comment:
Not a big deal, but `refsum()` reads slightly better to me.

Reply:
Done.